src/main/kotlin/thatline/localup/etcapi/service/ForecastService.kt

```
@@ -0,0 +1,124 @@
+        sync = true
+    )
+    fun findShortTermForecast(
+        sigunguCode: String,
+        dateTime: LocalDateTime = LocalDateTime.now(),
+    ): ShortTermForecastInformation {
+        val baseDate = dateTime.format(DateTimeUtil.DATETIME_FORMATTER_yyyyMMdd)
+        val baseTime = "0200"
+
+        // TODO: 로직 개선 필요
+        val tourApiLocation = TourApi.getTourApiAreaBySigunguCode(sigunguCode)
+            ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)